Components/Avatars/Avatars.css

```diff
@@ -0,0 +1,172 @@
+.avatars {
+  margin: 5rem 2rem;
+}
+
+.avatar {
+  display: flex;
+  gap: 5rem;
+  margin: 3rem;
+  width: 80%;
+  align-items: flex-end;
+}
+
+.radius-50 {
+  border-radius: 50%;
+}
+
+img {
+  width: 100%;
+  aspect-ratio: 1;
+}
+
+.placeholder-face {
+  height: 40%;
+  aspect-ratio: 1;
+  border: 0.1875rem solid hsl(var(--clr-dark));
+  border-radius: 50%;
+  background: hsl(var(--clr-white));
+  z-index: 999;
```

Comment:
change the z-index

Reply:
this is there for the placeholder face to stack above placeholder hands